### ISSUE TITLE
_.unique doesn't use iteratee when isSorted=true

### ISSUE THREAD

**Issue by DzenisevichK** [jashkenas/underscore#2025]
This test failed with latest version but it works with 1.6:

```js
deepEqual(_.uniq([{id: 1, val: 1}, {id: 1, val: 2}, {id: 2, val: 2}, {id: 3, val: 3}], true, "id"), [{id: 1, val: 1}, {id: 2, val: 2}, {id: 3, val: 3}], true);
```

**Comment by DzenisevichK** [comment]
I see that this already fixed in `master` version...

**Comment by megawac** [comment]
Yep, fixed in #1867
